hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -4648,6 +4650,29 @@ public List<SnapshotDiffJob> listSnapshotDiffJobs(String volume,
         bucket, jobStatus, listAll);
   }
 
+  public String printCompactionLogDag(String fileName,
+                                      String graphType)
+      throws IOException {
+
```

Comment:
Add a check here to allow only Ozone admins to run this.

Reply:
Do we need this check explicitly?

Reply:
Yes. Otherwise anyone can call this and fill up the `/tmp`

Reply:
I'll raise a PR addressing this comment and @errose28's concern.